packages/webdriverio/src/commands/browser/uploadFile.js

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * Uploads a file to the selenium server by using the [`file`](/api/protocol/file.html) command. Note that
```

Comment:
`Note that this command might not be supported as it is an undocumented Selenium command.`
This is part still true?

Reply:
Actually we should mention here that it is only supported in Chrome and using Selenium standalone